<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>unlockFrom</class>
 <widget class="QWidget" name="unlockFrom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>650</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:2px solid #0fbacd;
border-top-right-radius: 15px;
border-bottom-left-radius: 15px;
color:rgb(255,251,240);
background:rgba(29, 165, 219, 0.3);
font-size:22px;
}</string>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-size:32px;
color:rgb(29,165,219);
}</string>
   </property>
   <property name="text">
    <string>Unlock Advanced feature</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelUnlockCode">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>240</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-size:26px;
color:rgba(29,165,219,1.0);
};</string>
   </property>
   <property name="text">
    <string>Unlock Code:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCode">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>240</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-size:26px;
color:rgba(29,165,219,1.0);
};</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCode">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>320</y>
     <width>521</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
font-size:22px;
color:rgb(255,251,240);
background:rgba(0,0,0,0);
border:1px solid rgb(29,165,219);
};</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>22</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConfirm">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>400</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:2px solid #0fbacd;
border-top-right-radius: 15px;
border-bottom-left-radius: 15px;
color:rgb(255,251,240);
background:rgba(29, 165, 219, 0.3);
font-size:22px;
}</string>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSelMaster">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>171</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Master</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSelFactory">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>171</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Factory</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelUnlockSelection">
   <property name="geometry">
    <rect>
     <x>449</x>
     <y>180</y>
     <width>141</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-size:28px;
color:rgb(29, 165, 219);
}</string>
   </property>
   <property name="text">
    <string>Unlock Function:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSelSlave">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>171</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Slave</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
